### ISSUE TITLE
Invalid Aria attribute (Aria-activedescendant) is defined for text entered  in ckeditor.

### ISSUE THREAD

**Issue by Sukesh-Gundoji** [ckeditor/ckeditor4#3820]
Actual:
Invalid Aria attribute (aria-activedescendant) is defined for text entered  in "Rich Text editor field" in editor.
 
Expected:
Valid Aria attribute should be defined for text entered  in "Rich Text editor field" in editor. i.e aria-activedescendant should refer a ID value.

**Comment by Comandeer** [comment]
Can you provide more details? CKEditor 4 does not add any ARIA attributes to the inserted content.

**Comment by Dumluregn** [comment]
Hi, It's been a while since we last heard from you. We're closing this issue for now. Still, feel free provide us requested feedback, so that we can reopen this issue.
